src/main/java/uk/gov/pay/directdebit/mandate/api/CreateMandateResponse.java

```diff
@@ -45,6 +46,9 @@
 
     @JsonProperty("description")
     private String description;
+    
+    @JsonProperty("payment_provider")
+    private String paymentProvider = PaymentProvider.GOCARDLESS.toString().toLowerCase();
```

Comment:
Can't we get the actual value from the gateway account when we create this response?

Reply:
We could but we don't really have to. it's TDD and as long as the test passes we're good. There isn't going to be another direct debit payment provider anytime soon.

Reply:
We do have another provider: sandbox.

Reply:
sorry, you're right. i've fixed this now.